packages/core/src/types.ts

```diff
@@ -99,6 +99,12 @@ export type MadConfig<ParamList extends ParamListBase | void = void> = {
      * @see https://portal.azure.com/#@StatoilSRM.onmicrosoft.com/resource/subscriptions/2ad1b087-ffb8-4cf3-bcc2-8caeebfcd3f3/resourceGroups/mad-test/providers/Microsoft.Storage/storageAccounts/madtest62alhixjcodkm/storagebrowser
      */
     serviceNow?: EnvironmentValues<string>;
+    /**
+     * Experimental features can be toggled to activate new functionality that are in beta.
+     */
+    experimental: {
+        useExpoAuthSession?: boolean;
+    };
 };
 
```

Comment:
Do we re-export the correct functions when toggling this feature?

Reply:
im not sure if you can conditionally export packages based on a boolean. But i added export of Expo auth session from mad-core which was missing